lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -44,9 +47,12 @@ def current_shopify_session
           cookies: { cookie_name => cookies.encrypted[cookie_name] },
           is_online: user_session_expected?,
         )
+        # Rails.logger.debug("ShopifyApp - Loaded current shopify session")
```

Comment:
This was causing an error when loading the products. Not really sure how that relates to a log message.

```
ShopifyApp - Loaded current shopify session
Completed 500 Internal Server Error in 10ms (ActiveRecord: 1.9ms | Allocations: 5484)


  
NoMethodError (undefined method `shopify_session_id' for 44:Integer):
  
/Users/klenotiw/src/github.com/Shopify/shopify_app/lib/shopify_app/controller_concerns/login_protection.rb:223:in `session_id_conflicts_with_params'
/Users/klenotiw/src/github.com/Shopify/shopify_app/lib/shopify_app/controller_concerns/login_protection.rb:58:in `login_again_if_different_user_or_shop'
activesupport (7.0.3.1) lib/active_support/callbacks.rb:400:in `block in make_lambda'
```